public_dropin_notebook_environments/README.md

```diff
@@ -12,7 +12,7 @@ In this repository, we provide several example environments that you can use and
 * [Python 3.11 Notebook Environment](python311_notebook)
 * [Python 3.8 + Snowflake Notebook Environment](python38_snowflake_notebook)
 * [Python 3.9 Notebook Environment for GPU](python39_notebook_gpu)
-* [Python 3.9 Notebook Environment for GPU with Terraform](python39_notebook_gpu_tf)
+* [Python 3.9 Notebook Environment for GPU with Tensorflow](python39_notebook_gpu_tf)
```

Comment:
@lyapun this is how I recognized you are a software engineer not a data scientist 😄

Reply:
lol))
sorry (

Reply:
@roma-glushko could you also update it here: https://github.com/datarobot/datarobot-user-models/blob/master/public_dropin_notebook_environments/python39_notebook_gpu_tf/env_info.json#L3

I forgot to specify that it's tensorflow image in the env name

Reply:
Yes, will do!